try:
    a = 5
    b = 0
    c = a / b
except Exception as ex:
    print('다음과 같은 에러가 발생했습니다: {}'.format(ex))
